### DISCUSSION TITLE
@remix-run/server-runtime dependency

### DISCUSSION THREAD

**Discussion by noahsb** [sergiodxa/remix-auth#228] [Q&A] [answered]
I see `@remix-run/server-runtime` is listed as a peer dependency of the project, is this valid? I haven't had any problems using it without this library. In addition, I see the [remix auth form example](https://github.com/remix-run/examples/blob/main/remix-auth-form/package.json) does not have this package included. Curious if I should be installing this as a dependency? Thanks!!

**Comment by sergiodxa** [comment, marked as answer]
Server runtime is a dependency of the /node or /deno or /cloudflare packages, you don’t need to install it because it’s already installed. The reason is marked as peer is to use what your project has installed already.
